ci/packages/raspberry.go

```diff
@@ -117,6 +117,17 @@ func configureRaspbianImage(raspbianImagePath string) error {
 			log.Warn(err)
 		}
 	}()
+	err = shell.NewCmdf("sudo sed -i 's/^/#/g' %s", raspbianMountPoint+"/etc/ld.so.preload").Run()
+	if err != nil {
+		return err
+	}
+	defer func() {
+		err = shell.NewCmdf("sudo sed -i 's/^#//g' %s", raspbianMountPoint+"/etc/ld.so.preload").Run()
```

Comment:
double call for the same command?

Reply:
No, one comments out lines in preload, the deferred func restores the original.